packages/format-message/README.md

```diff
@@ -260,6 +260,27 @@ produces the same tree as
 </div>
 ```
 
+You can also provide an Object and use tag names that match the object's property names
```

Comment:
If we are going to have a restriction on the valid tag names, we should call it out here.